test/puppetlabs/puppetdb/amq_migration_test.clj

```diff
@@ -96,7 +96,7 @@
 
 (deftest shovel-upgrade-test
   (testing "an upgrade with existing catalogs"
-    (with-log-suppressed-unless-notable tlog/critical-errors
+    (tlog/with-log-suppressed-unless-notable log-utils/notable-pdb-event?
```

Comment:
Hmm, was this change intentional (here and below)?

Reply:
Ah yes, that's a mistake. I initially thought critical-errors was gone after the tk migration. But I brought it back